### PR TITLE
line numbers, horiz scrolling

### PR DESCRIPTION
**Pull request by Rick-Anderson**
per https://github.com/aspnet/Docs/issues/1054
@ardalis @danroth27 @shirhatti please review

### CODE REVIEW

File: aspnet/contribute/style-guide.rst [hunk 1]
```diff
@@ -126,7 +126,7 @@ Numbered lists can start with a number, or they can be autonumbered by starting
 Source Code
 ^^^^^^^^^^^
 
-Source code is very commonly included in these articles. Images should never be used to display source code - instead use ``code-block`` or ``literalinclude``. You can refer to it using the ``code-block`` element, which must be declared precisely as shown, including spaces, blank lines, and indentation:
+Source code is very commonly included in these articles. Images should never be used to display source code. Use ``literalinclude`` generally or ``code-block`` for snippets not included in the sample project (``literalinclude`` is strongly preferred). You can refer to it using the ``code-block`` element, which must be declared precisely as shown, including spaces, blank lines, and indentation:
```
Comment (shirhatti) [inline aspnet/contribute/style-guide.rst]: > Use  literalinclude  generally or  code-block  for snippets not included in the sample project ( literalinclude  is  strongly preferred). 

Sounds weird

Comment (ardalis) [inline aspnet/contribute/style-guide.rst]: Prefer ``literalinclude`` for most code samples. Reserve ``code-block`` for small snippets that are not included in the sample project.

File: aspnet/contribute/style-guide.rst [hunk 2]
```diff
@@ -178,6 +178,8 @@ This results in:
 		// Foo all the things!
 	}
 
+.. note:: One the ``emphasize-lines`` is determined, remove ``:linenos:``. When updating a doc, remove all occurrences of ``:linenos:``.
```
Comment (danroth27) [inline aspnet/contribute/style-guide.rst]: One -> Once

**Comment by danroth27** [comment]
Replaced with https://github.com/aspnet/Docs/pull/1168

**Pull request closed without merging**